test/cron_parser_test.dart

```diff
@@ -60,17 +61,9 @@ void main() {
         throwsA(TypeMatcher<AssertionError>()));
   });
 
-  test(
-      'Cron().parse() throws exception if current has been called before first next call',
-      () {
-    var cronIterator = Cron().parse("* * * * *", "Europe/Berlin");
-    expect(
-        () => cronIterator.current(), throwsA(TypeMatcher<AssertionError>()));
-  });
```

Comment:
@rbubke See my comment above. Therefore, I needed also to remove this test.

Reply:
Please leave it. it makes sense.

Reply:
Added the test again with ae6252bc6b4c5ae2005819a2cf03f4eef2c533cd ✅  I also adjusted the title of the test.